aux_ui/.env

```diff
@@ -0,0 +1,2 @@
+CLIENT_ID=316d39c75e9c4e13a93c937f6f6b7477
```

Comment:
spoop

Reply:
Removed cached env, shouldn't be tracked anymore!